scarb/src/bin/scarb/args.rs

```diff
@@ -570,6 +570,10 @@ pub struct LintArgs {
     /// Should fail on any warning.
     #[arg(short, long, default_value_t = false, env = "SCARB_LINT_DENY_WARNINGS")]
     pub deny_warnings: bool,
+
+    /// Path to a file or directory to lint. If provided, only this file or directory will be linted.
+    #[clap(value_name = "PATH")]
```

Comment:
```suggestion
    #[clap(value_name = "SCARB_FILE_PATH")]
```
wdyt? 
Overriding PATH may have unexpected effects. 
You can also change it in FmtArgs 😅

Reply:
`TARGET` is iffy, as it can easily be confused with Scarb target dir (https://docs.swmansion.com/scarb/docs/writing-extensions/subcommands.html#environment-variables) or even Scarb targets  (https://docs.swmansion.com/scarb/docs/reference/targets.html#targets) 😅 

Maybe `SCARB_ACTION_PATH` or smth? 🤔